### ISSUE TITLE
Single room saves

### ISSUE THREAD

**Issue by Airomis** [eccentricdevotion/TARDIS#343]
It would be nice to have the ability to save single custom rooms. Just like when we save modified tardis control rooms but this would be limited to a single use. Once the room is scanned the room will be auto jettison from the tardis. Then allowed to be grown in a different place. Once growing is finished the scamatica file for the room is deleted.

**Comment by eccentricdevotion** [comment]
Maybe same issue as with custom rooms #344
